stryker.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+  "files": [
+    "src/**/*.ts",
+    "test/**/*.ts"
+  ],
+  "mutate": [
+    "src/**/*.ts"
+  ],
+  "mutator": "typescript",
```

Comment:
what's the time or mutation/tests number budget here? In theory it could run forever but there must be a limit in place.

Reply:
https://github.com/stryker-mutator/stryker/tree/master/packages/core#timeoutms-number

Reply:
What I mean is how to make it generate more mutations, but the code may be too simple for that. Adding a couple of operations in `src/app.ts` raises the number of mutants:
```
Ran 1.60 tests per mutant on average.
----------|---------|----------|-----------|------------|----------|---------|
File      | % score | # killed | # timeout | # survived | # no cov | # error |
----------|---------|----------|-----------|------------|----------|---------|
All files |   25.00 |        1 |         0 |          3 |        0 |       1 |
 app.ts   |   25.00 |        1 |         0 |          3 |        0 |       1 |
----------|---------|----------|-----------|------------|----------|---------|
14:03:00 (17) ERROR MutationTestReportCalculator Final mutation score 25.00 under breaking threshold 100, setting exit code to 
1 (failure).
```

Reply:
https://github.com/libero/article-store/pull/167 shows more usage.